src/main/java/uk/gov/hmcts/darts/common/entity/MediaLinkedCaseEntity.java

```diff
@@ -22,7 +23,7 @@
 @EqualsAndHashCode(callSuper = false)
 @NoArgsConstructor
 @AllArgsConstructor
-public class MediaLinkedCaseEntity {
+public class MediaLinkedCaseEntity extends MandatoryCreatedBaseEntity {
```

Comment:
As `created_ts` and `created_by` are not mandatory per the DB schema, change to:

```suggestion
public class MediaLinkedCaseEntity extends CreatedBaseEntity {
```